### PR TITLE
docs: added 'Business Development Manager' career

### LINKED ISSUE #134: 职位： 商务拓展经理
------------------------职位信息如下：

岗位职责：
1、根据公司业务发展需求，寻找、挖掘有利于公司的合作资源，评估资源的可合作性；
2、负责对商务客户的管理和维护，独立高效地完成商务谈判、跟进、签约、合作落地、合同款项收付、客户关系维护等工作，达成业绩指标；
3、负责商务客户的维护，定期收集商业产品端的受众反馈，及时反馈给公司进行产品调整优化；
4、关注商务客户的动销情况，销售数据收集统计与分析。收集客户需求，为客户提供专业支持与服务，针对其销售、运营情况，提出合理化建议；
5、完成上级安排的其他工作。

任职要求：
1、本科及以上学历，具备良好远程协作能力，能有效的利用渠道资源；
2、了解开源文化，熟悉 API 网关、负载均衡产品等产品更佳；
3、优秀的商务谈判能力、协调能力和营销能力，能够独立完成营销工作。具备一定的抗压能力，有团队合作意识，有较强的执行力；
4、处事灵活，工作细致，逻辑思维能力强，具备处理突发事件能力。

### PR DESCRIPTION
**Pull request by guoqqqi**
resolve #134 
Added Business Development Manager position to the official job page.

### CODE REVIEW

**Comment by guoqqqi** [comment]
@membphis 我为“商务拓展经理”卡片设置的类为：商务团队，这样是否可行、见图：
image

**Comment by guoqqqi** [comment]
@juzhiyuan  @LiteSun  有时间了请帮我检查一下英文版的翻译😊😊

File: _posts/careers/en-US/business-development-manager.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Excellent business negotiation, coordination and marketing skills, able to complete marketing work independently. Ability to resist pressure, a sense of teamwork and strong execution.
+- Flexible, meticulous, logical thinker with the ability to deal with emergencies.
+
+## Additional
+
+We're a remote work company with employees in 6 cities across China, and we do all of our collaboration through GitHub, Slack and Google Docs.
+
+## Contact
+
+[wenming@api7.ai](mailto:wenming@api7.ai)
```
Comment (juzhiyuan) [inline _posts/careers/en-US/business-development-manager.md:27]: We will use this one

```suggestion
[support@api7.ai](mailto:support@api7.ai)
```

File: _posts/careers/zh-CN/business-development-manager.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+date: 2021-06-22 21:52:00
+---
+
+### 工作内容
+
+1. 根据公司业务发展需求，寻找、挖掘有利于公司的合作资源，评估资源的可合作性；
+2. 负责对商务客户的管理和维护，独立高效地完成商务谈判、跟进、签约、合作落地、合同款项收付、客户关系维护等工作，达成业绩指标；
+3. 负责商务客户的维护，定期收集商业产品端的受众反馈，及时反馈给公司进行产品调整优化；
+4. 关注商务客户的动销情况，销售数据收集统计与分析。收集客户需求，为客户提供专业支持与服务，针对其销售、运营情况，提出合理化建议；
+5. 完成上级安排的其他工作。
```
Comment (juzhiyuan) [inline _posts/careers/zh-CN/business-development-manager.md:12]:
```suggestion
5. 完成上级安排的其它工作。
```

File: _posts/careers/zh-CN/business-development-manager.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+### 岗位要求
+
+1. 本科及以上学历，具备良好远程协作能力，能有效的利用渠道资源；
+2. 了解开源文化，熟悉 API 网关、负载均衡产品等产品更佳；
+3. 优秀的商务谈判能力、协调能力和营销能力，能够独立完成营销工作。具备一定的抗压能力，有团队合作意识，有较强的执行力；
+4. 处事灵活，工作细致，逻辑思维能力强，具备处理突发事件能力。
+
+### 附加信息
+
+薪酬超过行业的平均水平；
```
Comment (juzhiyuan) [inline _posts/careers/zh-CN/business-development-manager.md:23]: 请添加序号

File: _posts/careers/zh-CN/business-development-manager.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+一个可以平衡事业和生活的工作环境；
+
+年龄仅仅是一个数字。不管您刚刚大学毕业还是已有家室，我们只为您的能力而来；
+
+我们是一家分布式的公司，员工分布在国内 6 个城市，通过 GitHub、Slack、谷歌文档、腾讯会议完成全部的协作。
+
+### 联系方式
+
+[wenming@api7.ai](mailto:wenming@api7.ai)
```
Comment (juzhiyuan) [inline _posts/careers/zh-CN/business-development-manager.md:39]: same here

File: _posts/careers/en-US/business-development-manager.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Business Development Manager
```
Comment (juzhiyuan) [inline _posts/careers/en-US/business-development-manager.md:2]: Not sure if it's correct here

File: _posts/careers/en-US/business-development-manager.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+date: 2021-06-22 21:52:00
+---
+
+## Content
+
+- According to the company's business development needs, find and explore cooperation resources that are beneficial to the company, and evaluate the cooperability of resources.
```
Comment (juzhiyuan) [inline _posts/careers/en-US/business-development-manager.md:8]: Hi @iamayushdas, is this file's contents clear to you?

File: _posts/careers/en-US/business-development-manager.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+title: Business Development Manager
+date: 2021-06-22 21:52:00
+---
+
+## Content
```
Comment (iamayushdas) [inline _posts/careers/en-US/business-development-manager.md:6]: `Content` heading should be replaced with `Responsibilities` , to be more precise,

content of this looks good to me for overseas/foreign people too

**Review by iamayushdas — CHANGES_REQUESTED** [review]
Just a change in Content heading

File: _posts/careers/en-US/business-development-manager.md (hunk 1 above)
Comment (guoqqqi) [reply _posts/careers/en-US/business-development-manager.md:27]: OK, I see that the other files are still at that address and I will use a new PR to optimize the other files.

File: _posts/careers/en-US/business-development-manager.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
+---
+title: Business Development Manager
+date: 2021-06-22 21:52:00
+---
+
+## Responsibilities
+
+- According to the company's business development needs, find and explore cooperation resources that are beneficial to the company, and evaluate the cooperability of resources.
+- Responsible for the management and maintenance of business customers, independently and efficiently complete business negotiations, follow-up, contract signing, cooperation landing, contract payment collection and payment, customer relationship maintenance and other work to achieve performance targets.
```
Comment (liuxiran) [inline _posts/careers/en-US/business-development-manager.md:9]:
```suggestion
- Responsible for the management and maintenance of business customers, independently and efficiently complete business negotiations, follow-up, contract signing, cooperation landing, contract payment collection and payment, customer relationship maintenance and other works to achieve performance targets.
```

File: _posts/careers/en-US/business-development-manager.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+- Responsible for the maintenance of business customers, regular collection of audience feedback on the commercial product side and timely feedback to the company for product adjustment and optimization.
```
Comment (liuxiran) [inline _posts/careers/en-US/business-development-manager.md:10]:
```suggestion
- Responsible for the maintenance of business customers, regular collection of audience feedbacks on the commercial product side and timely feedback to the company for product adjustment and optimization.
```

File: _posts/careers/en-US/business-development-manager.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+- Pay attention to the dynamic sales situation of business customers, sales data collection statistics and analysis. Collecting customer needs, providing professional support and services to customers, and making rationalized suggestions for their sales and operational situations.
```
Comment (liuxiran) [inline _posts/careers/en-US/business-development-manager.md:11]:
```suggestion
- Follow the dynamic sales situation of business customers, collect and analyze sales statistics. Collecting customer needs, providing professional support and services to customers, and making rationalized suggestions for their sales and operational situations.
```

**Review by liuxiran — APPROVED** [review]
LGTM

**Pull request merged**